src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -248,7 +265,7 @@ private static void AddResponseHeader(HttpResponseMessage response, KeyValuePair
                         }
                         break;
                     case "content-md5":
-                        response.Content.Headers.ContentMD5 = header.Value.Value<byte[]>();
+                        response.Content.Headers.ContentMD5 = header.Value as byte[];
```

Comment:
A string here would be valid as well. (I believe I had a similar comment on @mamaso's PR)